packages/contentstack-export-to-csv/test/unit/commands/export-to-csv.test.js

```diff
@@ -1,219 +1,289 @@
-/* eslint-disable no-undef */
-const { describe, it, beforeEach, afterEach } = require('mocha');
-const ExportToCsvCommand = require('../../../src/commands/cm/export-to-csv');
-const { stub, assert } = require('sinon');
-const { config } = require('dotenv');
+const fs = require('fs');
+const { test, expect } = require('@oclif/test');
+const { join } = require('path');
+const { PassThrough } = require('stream');
 const inquirer = require('inquirer');
-const { cliux } = require('@contentstack/cli-utilities');
+const checkboxPlus = require('inquirer-checkbox-plus-prompt');
+const { cliux, configHandler } = require('@contentstack/cli-utilities');
 
-config();
+const mockData = require('../../mock-data/common.mock.json');
 
-describe('Export to csv command with action = entries', () => {
-  let inquireStub;
-  let errorStub;
-  let consoleLogStub;
+const { cma } = configHandler.get('region');
+const directory = join(process.cwd(), 'data');
+const taxonomyFileName = join(process.cwd(), 'data', mockData.stacks[0].name);
 
-  beforeEach(() => {
-    inquireStub = stub(inquirer, 'prompt');
-    errorStub = stub(cliux, 'error');
-    consoleLogStub = stub(cliux, 'print');
-  });
-
-  afterEach(() => {
-    inquireStub.restore();
-    errorStub.restore();
-    consoleLogStub.restore();
-  });
-
-  it('Should ask for action when action is not passed (entries or users)', async () => {
-    await ExportToCsvCommand.run([]);
-    assert.calledOnce(inquireStub);
-  });
-
-  it('Should ask for org when org is not passed', async () => {
-    const args = ['--action', 'entries'];
-    await ExportToCsvCommand.run(args);
-    assert.calledOnce(inquireStub);
-  });
+describe('export-to-csv with action taxonomies', () => {
+  if (!fs.existsSync(directory)) fs.mkdirSync(directory);
```

Comment:
Why are we creating a folder in the unit test case.?

Reply:
@antonyagustine removed it.